rpa-builder/6.2/modules/ROOT/pages/activity-library-explorer.adoc

```diff
@@ -0,0 +1,52 @@
+= Activity Library Explorer 
+
+In the Activity Library Explorer you can view all the activity templates assigned to your organization. All users in your organization with the privilege *Activity Library Administration* have access to the same Activity Library.
+
+If another user adds an Activity, you see the change after refreshing the Activity Library Explorer.
```

Comment:
Is the init cap necessary here? Maybe since it's not specifically referring to the UI, it can be "activity"?